### ISSUE TITLE
Tremendously slow flow centrality

### ISSUE THREAD

**Issue by jefferis** [flyconnectome/hemibrainr#18]
testing with bodyid "1702323386"

This area is one of the main culprits. 

https://github.com/flyconnectome/hemibrainr/blob/dc11c95da706025d72f8f35514dfc0d95e30f261/R/hemibrain_manual_split.R#L362-L370

@alexanderbates if you are iterating with sapply/for etc over igraph node ids, there is a problem.

```r
Browse[2]> system.time(a <- unlist(igraph::ego(n, nodes=s.d, mode='in', mindist = 1)))
   user  system elapsed 
  0.015   0.039   0.054 
Browse[2]> system.time(b <- sapply(s.d,function(s) igraph::neighbors(n, v=s, mode = c("in"))))
   user  system elapsed 
 20.024   1.102  21.162 
Browse[2]> all.equal(a,b)
[1] "names for current but not for target"
```

For the timings in line 1 there, identical results (bar names on the result verctor which are required in 50 ms vs 20s.

**Comment by jefferis** [comment]
Closed by #19

**Comment by alexanderbates** [comment]
Noted thank you
